apps/server/src/modules/lesson/service/lesson.service.spec.ts

```diff
@@ -90,7 +90,7 @@ describe('LessonService', () => {
 	describe('findAllLessonsByUserId', () => {
 		describe('when finding by userId', () => {
 			const setup = () => {
-				const userId = new ObjectId().toHexString();
+				const userId = new ObjectId();
```

Comment:
const userId = new ObjectId().toHexString();

Reply:
@WojciechGrancow

Reply:
but user in ComponentProperties is ObjectId type